forum/webapp/src/main/webapp/templates/forum/webui/UIBreadcumbs.gtmpl

```diff
@@ -73,6 +73,7 @@
 	  <%String toolTip = uicomponent.getToolTip();
 		if(checkPath.equals(ForumUtils.FIELD_SEARCHFORUM_LABEL) == false && toolTip.equals(ForumUtils.TAG) == false){
 			toolTip = _ctx.appRes("UIBreadcumbs.title." + toolTip);
+		link = ForumUtils.createdForumLink(uicomponent.getType(id), id, uicomponent.checkLinkPrivate(id));

```

Comment:
could you rename the property link to avoid confusion !
You can use forumPermalink for example